modules/ssl/files/ssl-certificate.py

```diff
@@ -107,9 +108,13 @@ def generate_letsencrypt_certificate(self):
                 print('Generating Wildcard SSL certificate with LetsEncrypt')
 
             if self.no_existing_key:
-                os.system(f'/usr/bin/certbot --force-renewal --expand --no-verify-ssl certonly --manual --preferred-challenges dns-01 {self.overwrite} -d {self.domain} {self.secondary_domain}')
+                out = os.system(f'/usr/bin/certbot --force-renewal --expand --no-verify-ssl certonly --manual --preferred-challenges dns-01 {self.overwrite} -d {self.domain} {self.secondary_domain}')
+                if out != 0:
+                     sys.exit("Error: Certbot failed (either the domain isn't pointed or we are being rate limited)")
```

Comment:
We need to know the exact rate limit error. Different errors from this is different lengths for the rate limit, to know how long the rate limit lasts, we don't a generic catch-all error.
Including below also.

Reply:
assuming -q wasn't used we would still see the output from certbot